src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -16,6 +16,7 @@
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Emit;
 using Roslyn.Utilities;
+using static Microsoft.CodeAnalysis.CSharp.Symbols.SynthesizedEntryPointSymbol;
```

Comment:
I mean, yeah, it does a member that is being used statically.  However, I could just fully-qualify that use.